eva-stats/stats_variant_warehouse.py

```diff
@@ -45,7 +46,7 @@ def get_mongo_primary_host_and_port(mongo_hosts_and_ports):
 
 
 def get_from_variant_warehouse(mongo_handle, metadata_handle, projects):
-    projects_evapro = "('" + projects.replace(",", "','") + "')"
+    projects_evapro = str(projects).replace("[", "(").replace("]", ")")
```

Comment:
This ties us to the list representation of projects and not any iterable. Suggest `"[{0}]".format(",".join(projects))` instead.

Reply:
Thanks, changed